chat-widget/src/components/livechatwidget/common/endChat.ts

```diff
@@ -33,6 +33,11 @@ const prepareEndChat = async (props: ILiveChatWidgetProps, chatSDK: any, state:
             return;
         }
 
+        // Register post chat participant type
+        if (conversationDetails?.participantType === ParticipantType.Bot || conversationDetails?.participantType === ParticipantType.User) {
+            dispatch({ type: LiveChatWidgetActionType.SET_POST_CHAT_PARTICIPANT_TYPE, payload: conversationDetails?.participantType });
+        }
```

Comment:
it seems like the if block is irrelevant, only exists 2 types of participant, bot or user,

Reply:
conversationDetails?.participantType can be null, or something else. Added for backward compatibility. 

Removing it is dangerous